examples/docs_beta_snippets/docs_beta_snippets/guides/external-systems/databases/use-envvars.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+import pandas as pd
+from dagster_snowflake import SnowflakeResource
+from snowflake.connector.pandas_tools import write_pandas
+
+import dagster as dg
+
+# highlight-start
+# Define `local` and `production` versions of the Snowflake resource
+resources = {
+    "local": {
+        "iris_db": SnowflakeResource(
+            # Retrieve dev credentials with environment variables
+            user=dg.EnvVar("DEV_SNOWFLAKE_USER"),
```

Comment:
BK is failing because these env vars aren't sourced - what's the right way to handle this?

Reply:
Suggested a fix here to fix unit tests:
https://github.com/dagster-io/dagster/pull/24506/files#r1763336244